docs/extension.rst

```diff
@@ -345,8 +343,14 @@ is discussed below.
 - ``build-extension: true`` instructs flatpak to build an extension.
 - ``separate-locales: false`` disables creating a ``.Locale`` extension
   for this extension.
-- ``appstream-compose: false`` disables running ``appstream-compose``.
-  This is run manually in the build or cleanup stage.
+
+With recent versions of flatpak-builder and appstreamcli, it is no longer
+required for extensions to manually compose the metadata. Flatpak builder
+and appstreamcli will handle this out of the box.
```

Comment:
we need to make sure it does for extensions build out of the same source. There are a couple Audio apps that comes with plugin builds.